catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/ResourceOperationsTest.java

```diff
@@ -434,11 +417,34 @@ public void testGetResourceInfoWithFragmentAndMultipleResults() throws Exception
     ResourceOperations.ResourceInfo resourceInfo =
         resourceOperationsSpy.getResourceInfo(
             resourceRequestMock, "site", isEnterprise, federatedSite, requestProperties, false);
-    verify(resourceOperationsSpy)
-        .createPropertyStartsWithQuery(Metacard.RESOURCE_URI, "bobUri:test");
```

Comment:
These verify statements were being used to make sure that the fragment was removed from the uri first before making the query. Since we don't do that anymore these statements are no longer needed.